cmd/influx/main.go

```diff
@@ -160,18 +174,102 @@ func main() {
 				Usage: "Check the InfluxDB /health endpoint",
 				Flags: commonFlags,
```

Comment:
I don't think you need the auth token to hit the `/health` endpoint: https://github.com/influxdata/influxdb/blob/7bde3413b3801bdb15095ddb492e4ea2e916fc28/http/handler.go#L123